src/js/components/Tabs/__tests__/Tabs-test.tsx

```diff
@@ -193,7 +193,7 @@ describe('Tabs', () => {
 
     expect(container.firstChild).toMatchSnapshot();
 
-    const disabledTab = getByText('Disabled Tab').parentElement;
+    const disabledTab = getByText('Disabled Tab').parentElement as HTMLElement;
```

Comment:
I prefer a non-null assertion operator in the context of tests, as it uses less space and more explicitly addresses null types.